packages/interval/interval.1.3/files/META

```diff
@@ -0,0 +1,6 @@
+name = "interval"
```

Comment:
I agree.  I am discussing with the authors but in order not to delay the initial release, I put it in `files/`.